service-print/src/main/java/org/oskari/print/util/StyleUtil.java

```diff
@@ -59,7 +56,7 @@
 
     public static PDPrintStyle getLineStyle (JSONObject oskariStyle) {
         PDPrintStyle style = new PDPrintStyle();
-        JSONObject stroke = JSONHelper.getJSONObject(oskariStyle, "stroke");
+        JSONObject stroke = JSONHelper.optJSONObject(oskariStyle, "stroke", () -> new JSONObject());
```

Comment:
Wonder if this would work/read better if PDPrintStyle had the default values itself and we could just return early instead of creating new objects to make the rest of the code work properly.